src/index.js

```diff
@@ -1 +1,42 @@
 import './style.css';
+
+async function sendData(url = '', data = {}) {
```

Comment:
Let's use the ES6 arrow function here instead of the `function` keyword ✊

Reply:
@bimbolabuari please explain what do you mean since the code is the same as the exercise offered by Microverse on mozilla.org

Reply:
Changes have been done - thanks